reactive/kubernetes_master.py

```diff
@@ -2250,11 +2250,19 @@ def configure_scheduler():
     scheduler_opts['profiling'] = 'false'
     scheduler_opts['config'] = kube_scheduler_config_path
 
+    scheduler_ver = get_version('kube-scheduler')
```

Comment:
FYI, we needed to switch the api version based on the k8s version.

kubescheduler.config.k8s.io/v1alpha1 was removed in 1.19 by kubernetes/kubernetes#89298.
kubescheduler.config.k8s.io/v1alpha2 was added in 1.18 by kubernetes/kubernetes#87628.